db/DB.js

```diff
@@ -78,7 +78,7 @@ const DB = {
   },
 
   // Used internally.
-  async _addEmailHash(sha1, email, verified = false) {
+  async _addEmailHash(sha1, email, signup_language, verified = false) {
```

Comment:
Nit: not sure if this should be `signupLanguage` to be consistent w/ `async addSubscriber()` implementation below on L120.

Reply:
This also made me look back at some kind of node ORM library. @pdehaan - do you know if any other Mozilla node projects are using https://bookshelfjs.org/ ?